inline_scan.sh

```diff
@@ -595,11 +576,7 @@ cleanup() {
         unset DOCKER_ID
     done
 
-    if [[ "${#IMAGE_FILES[@]}" -ge 1 ]] || [[ -f /tmp/sysdig/sysdig_output.log ]]; then
-        if [[ -d "/tmp/sysdig" ]]; then
-            rm -rf "/tmp/sysdig"
-        fi
-    fi
+    rm -rf ${VOLUME_PATH}
```

Comment:
```suggestion
    rm -rf "${VOLUME_PATH}"
```
consistency with other calls, also maybe a log line?